### PR TITLE
fix buildswift script

### PR DESCRIPTION
**Pull request by corpaul**
now also accepts 'false' values :D

### CODE REVIEW

File: scripts/buildswift.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 build () {
     # @CONF_OPTION DEBUG_SWIFT: Set to any value if you want to enable Swift's debug output. (default is disabled)
-    if [ -z "$DEBUG_SWIFT" ]; then
+    if [ "$DEBUG_SWIFT{,,}" == "false" ]; then
```
Comment (whirm) [inline scripts/buildswift.sh]: ${DEBUG_SWIFT,,}

File: scripts/buildswift.sh [hunk 2]
```diff
@@ -50,7 +50,7 @@ build () {
 
 }
 
-if [ ! -z "$BUILD_SWIFT" ]; then
+if [ "$BUILD_SWIFT{,,}" == "true" ]; then
```
Comment (whirm) [inline scripts/buildswift.sh]: ditto

File: scripts/buildswift.sh [hunk 3]
```diff
@@ -41,7 +41,7 @@
 
 build () {
     # @CONF_OPTION DEBUG_SWIFT: Set to any value if you want to enable Swift's debug output. (default is disabled)
```
Comment (whirm) [inline scripts/buildswift.sh]: Update the command line description and don't change the default behaviour.

**Comment by corpaul** [comment]
ready to merge

**Pull request merged**